backend/tests/index.test.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * @jest-environment node
+ */
+const request = require("supertest");
+const app = require("../app");
+const DataBase = require("../database");
+const DB = new DataBase();
+const expectedJSON = {
+  originalUrl: "http://www.github.com",
+  shorturlId: "vnFthnWBJ",
+  creationDate: "2021-03-07 10:39:11",
+  redirectCount: 0,
+};
+const expectedURL = {
+  url: "http://www.github.com",
+};
+const notRealURL = {
+  url: "www.facebokde.com",
+};
+
+beforeEach(async () => {
+  await DB.putBin([expectedJSON]);
+});
+
+describe("POST route", () => {
+  it("Should return the same shortURL every time", async () => {
+    const response = await request(app)
+      .post("/api/shorturl/new/")
+      .send(expectedURL);
+    expect.assertions(2);
+    // Is the status code 200
+    expect(response.status).toBe(200);
+
+    // Is Json created properly
+    expect(response.body).toEqual(expectedJSON);
+  });
```

Comment:
Where are you checking it returns the same URL?

Reply:
By creating a new URL post request and check if the short URL hasn't changed. In the beginning, I send only the URL of the website and expect to get the same result every time, the expectedJSON (line 8).